### PR TITLE
Linked the TFE table to help with manual configuration

### PR DESCRIPTION
**Pull request by henryk-rychlik**
### What

Added a new comment to Okta SSO config page. 

### Why

Names of links between Okta and Terraform differ, which is confusing during manual configuration.

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. HCP Terraform ).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/users-teams-organizations/single-sign-on/okta.mdx [hunk 1]
```diff
@@ -27,6 +27,9 @@ For more information on the listed features, visit the [Okta Glossary](https://h
 
 For information on configuring automated team mapping using Okta group membership, please see the [Team Mapping Configuration (Okta)](#team-mapping-configuration-okta-) section below.
 
+Creation of an Application on Okta side requires elevated privileges. For that reason customers sometimes create a workflow to provision it, leaving the end user with a need to configure the actual settings.
```
Comment (rkoron007) [inline website/docs/cloud-docs/users-teams-organizations/single-sign-on/okta.mdx:30]: Because this is a very particular scenario you are describing about setting up an Okta feature, rather than an HCP Terraform feature, it feels like it might be a better fit for a support article, rather than an additional to our documentation. In general, we want to try to avoid describing how other products work in our documentation as much as possible. Let me know what you think though! Happy to have a conversation about this if you disagree!

**Review by rkoron007 — CHANGES_REQUESTED** [review]
Left a comment, happy to continue this conversation though!

File: website/docs/cloud-docs/users-teams-organizations/single-sign-on/okta.mdx (hunk 1 above)
Comment (henryk-rychlik) [reply website/docs/cloud-docs/users-teams-organizations/single-sign-on/okta.mdx:30]: I understand. Let me rephrase a little bit then.
Configuration using Terraform Cloud application from Okta side is flawless and requires very few steps.
Some customers don't want to grant this level of access and they use a separate workflow to create a generic app in Okta. When you look at the table from step 4, Single sign on URL on our side is ACS Consumer in Okta. Customers tend to paste a wrong value in Okta because our config shows Single sign on URL as well.
To sum it up, they do not what's in the table, but Single sign on from HCP -> Single sign on in Okta.
This already exists in TFE docs, probably for exactly that reason. I can do the article if you still think it's the best route. Just wanted to make sure the why is clear.

Comment (rkoron007) [reply website/docs/cloud-docs/users-teams-organizations/single-sign-on/okta.mdx:30]: Thank you for the extra context, I really appreciate it! I do still think this would be better presented in a support article because it is such a particular use case and workflow that has more to do with Okta than on our own products. Thanks for looping me in though!
